web_src/js/index.js

```diff
@@ -319,7 +330,76 @@ function uploadFile(file, callback) {
 }
 
 function reload() {
-  window.location.reload();
+  if ($('#current-card-details').length > 0) {
+    reloadIssuesActions();
+  } else {
+    window.location.reload();
+  }
+}
+
+function reloadIssuesActions(data) {
+  if ($('#current-card-details').length === 0) {
+    window.location.reload();
+  }
+  if (!data || !data.url) {
+    data = $('body').data();
+    if (!data.url) {
+      data.url = `${window.location.href}/sidebar/true`;
+    }
+  } else {
+    $('body').data('url', data.url);
+  }
+  fetch(data.url, {
+    method: 'GET',
+    headers: {'X-Csrf-Token': csrf, 'Content-Type': 'text/html'}
+  })
+    .then((res) => {
```

Comment:
Please use async/await instead of `.then` chaining.

Reply:
This literally does the same thing with no downside when using then() in this case. Feel free to submit a conversion though and I will apply it.

Reply:
Yes, it's just code style and readability (less nesting). I can probably do some diffs later.